File: 백준/Bronze/16727. ICPC/ICPC.py
import sys
input=lambda:sys.stdin.readline().rstrip()
p1,s1=map(int,input().split())
s2,p2=map(int,input().split())
pt,st=p1+p2,s1+s2
if pt==st:
    print("Penalty" if s1==p2 else "Esteghlal" if s1>p2  else "Persepolis")
else:
    print("Esteghlal" if st>pt  else "Persepolis") 

File: 백준/Bronze/16727. ICPC/ICPC.cc
#include <iostream>
using namespace std;
int main(void) {
    int s1, s2, p1, p2, st, pt;
    cin >> p1 >> s1;
    cin >> s2 >> p2;
    st = s1 + s2;
    pt = p1 + p2;
    if (pt == st)
        cout << (s1 == p2 ? "Penalty" : (s1 > p2 ? "Esteghlal" : "Persepolis"));
    else
        cout << (st > pt ? "Esteghlal" : "Persepolis");
    return 0;
}